src/cdk/drag-drop/directives/drop-list.ts

```diff
@@ -256,7 +255,7 @@ export class CdkDropList<T = any> implements OnDestroy {
         if (typeof drop === 'string') {
           const correspondingDropList = CdkDropList._dropLists.find(list => list.id === drop);
 
-          if (!correspondingDropList && isDevMode()) {
+          if (!correspondingDropList && (typeof ngDevMode === 'undefined' || ngDevMode)) {
```

Comment:
Does removing `isDevMode()` mean that View Engine apps will have warnings/errors as in dev mode? It looks like View Engine `enableProdMode` does not set `ngDevMode` at all. Maybe it should?

Reply:
Ah, I assumed that it would. We would need to fix that since we still need to report these issues in ViewEngine.

Reply:
I believe that `enableProdMode` doesn't set the variable, because it's supposed to be replaced during build time. I don't think that this will be an issue for ViewEngine apps, because the majority of the current checks were running in production too. We have a handful of `isDevMode` ones, but they're sanity checks so it wouldn't really be a problem if they ran in production. The only one that may be a problem is the theme check in the common module since it has a `getComputedStyle` call so I'll add an exception for it.